gpt_json/gpt.py

```diff
@@ -112,9 +119,14 @@ def __init__(
                 self.max_tokens = 8192 - auto_trim_response_overhead
             elif "gpt-3.5" in self.model:
                 self.max_tokens = 4096 - auto_trim_response_overhead
+            elif "claud-v1" in self.model:
+                # todo confirm this number
```

Comment:
Seems like at least in their release notes they cite 9k as the previous limit? https://www.anthropic.com/index/100k-context-windows